deploy/crds/org_v1_che_cr.yaml

```diff
@@ -23,13 +23,13 @@ spec:
     # image:tag used in Devfile registry deployment
     devfileRegistryImage: 'quay.io/eclipse/che-devfile-registry:nightly'
     # image:tag used in plugin registry deployment
-    pluginRegistryImage: 'quay.io/eclipse/che-plugin-registry:nighlty'
+    pluginRegistryImage: 'quay.io/eclipse/che-plugin-registry:nightly'
     # defaults to `che`. When set to `codeready`, CodeReady Workspaces is deployed
     # the difference is in images, labels, exec commands
     cheFlavor: ''
     # specifies a custom cluster role to user for the Che workspaces
     # Uses the default roles if left blank.
-    cheWorkspaceClusterRole: ''
+    cheWorkspaceClusterRole: 'eclipse-codewind'
```

Comment:
Thank you @johnmcollier for this fix. Changes to this line and line 72 are included in the PR on purpose?

Reply:
Ugh, sorry!!!

Reply:
Removing now, my bad

Reply:
@l0rd Thanks for catching that, fixed it